src/main/scala/viper/gobra/reporting/StreamingReporter.scala

```diff
@@ -0,0 +1,24 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+
+package viper.gobra.reporting
+
+import com.typesafe.scalalogging.StrictLogging
+
+case class StreamingReporter(reporter: GobraReporter) extends GobraReporter with StrictLogging {
+  override val name: String = reporter.name
+
+  def report(msg: GobraMessage): Unit = {
+    msg match {
+      case m:GobraEntityFailureMessage => m.result match {
+        case VerifierResult.Failure(errors) => errors.foreach(err => logger.error(s"Error: ${err.formattedMessage}"))
```

Comment:
This way, we will only log the errors if we are using the `StreamingReporter`, which will probably not be the case with the IDE right? Could we do the reporting somewhere else, like in `BacktranslatingReporter`?

Reply:
Although this is correct, I don't think errors printed to StdOut should be of concern for the IDE